src/main/webapp/welcomeStyle.css

```diff
@@ -169,11 +186,12 @@ body {
   width: 25%;
   border-style: double;
   border-color: gray;
+  position: relative;
 }
 
 .panel-header {
   text-align: center;
-  font-size: 2.5em;
+  font-size: 3vw;
```

Comment:
Why are you using viewport width units here?

Reply:
I was using viewport units because the size of the fonts became disproportionate with smaller screen sizes. For instance the CHAP on the welcome page took up too much of the page and the panels below became much too squished. I usually use em units or pixels but in this context, the font sizes did not fit inside the widths of the panels when the window size changed which is why I used viewport sizes.